### DISCUSSION TITLE
Alerting and Actions (Kibana)

### DISCUSSION THREAD

**Discussion by TobMue** [telekom-security/tpotce#745] [General]
Has anyone had experience with Kibana's new [Alerting and Actions](https://www.elastic.co/guide/en/kibana/current/alerting-getting-started.html) functionality and tried using it along with T-POT?

For the use of T-POT as a production honeypot, this could be a very helpful way to be notified in case of an attack.

**Comment by t3chn0m4g3** [comment]
Alerting and Actions requires X-Pack which is part of a paid license, as such an integration into T-Pot is not possible.
